c_tests/tests/mapper_funcs_branches.c

```diff
@@ -0,0 +1,51 @@
+// Compiler:
```

Comment:
What's this?

Reply:
It's a `lang_tester` test. [It basically just tests that the compiler and run-time don't error at the moment.]